### ISSUE TITLE
payloads of NLMSG_DONE messages are not accessible 

### ISSUE THREAD

**Issue by yshui** [little-dude/netlink#160]

**Comment by little-dude** [comment]
Yeah. You mean in `netlink-proto` right? IIRC I did that on purpose. What problem does that cause exactly?

**Comment by stbuehler** [comment]
I think this can be closed in favor of #230 (although the #230 title doesn't mention DONE explicitly)
